### PR TITLE
Assisted Mode - RIP-218

### PR DESCRIPTION
**Pull request by autotelik**
https://eaflood.atlassian.net/browse/RIP-218

Set Fully Assisted when a BO user present at the Exemption state

### CODE REVIEW

File: app/models/flood_risk_engine/enrollment_exemption.rb [hunk 1]
```diff
@@ -22,6 +22,13 @@ class EnrollmentExemption < ActiveRecord::Base
       operator_failings: 1
     }
 
+    enum assistance_mode: {
+      unassisted: 0,
+      fully: 1,
+      partial: 2,
+      admin_intervention: 3
+    }
+
```
Comment (woodpigeon) [inline app/models/flood_risk_engine/enrollment_exemption.rb]: Just wondered if the `fully` and `partial` enum values were e.g. `fully_assisted` and `partially_assisted` then calling `enrollment_exemption.fully_assisted!` would have more meaning than `enrollment_exemption.fully!`

Comment (autotelik) [inline app/models/flood_risk_engine/enrollment_exemption.rb]: yeah been umming and ahhhing whether  to change them, but just went with WEX values n the end for simplicity ..only used in couple of places

**Comment by woodpigeon** [comment]
:ship:

File: app/models/flood_risk_engine/enrollment_exemption.rb (hunk 1 above)
Comment (autotelik) [inline app/models/flood_risk_engine/enrollment_exemption.rb]: I'm gonna change fully to fully_assisted .. does read much better

**Pull request merged**